README.md

```diff
@@ -40,43 +32,51 @@ Apart from the performance requirements, we also need a storage system
 to ensure multiple-level data safety and support fast data migration
 between data centers, automatic load balancing, and online partition split.
 
-After investigating the existing storage systems in the open source world,
-we could hardly find a suitable solution to satisfy all the requirements.
-So the journey of Pegasus begins.
+## Features
+
+- **Persistence of data**: Each write is replicated three-way to different ReplicaServers before responding to the client. Using PacificA protocol, Pegasus has the ability for strong consistent replication and membership changes.
+
+- **Automatic load balancing over ReplicaServers**: Load balancing is a builtin function of MetaServer, which manages the distribution of replicas. When the cluster is in an inbalance state, the administrator can invoke a simple rebalance command that automatically schedules the replica migration.
+
+- **Cold Backup**: Pegasus supports an extensible backup and restore mechanism to ensure data safety. The location of snapshot could be a distributed filesystem like HDFS or local filesystem. The snapshot storing in the filesystem can be further used for analysis based on [pegasus-spark](https://github.com/pegasus-kv/pegasus-spark).
+
+- **Eventually-consistent intra-datacenter replication**: This is a feature we called *duplication*. It allows a change made in the local cluster accesible after a short time period by the remote cluster. It help achieving higher availability of your service and gaining better performance by accessing only local cluster.
 
 ## To start using Pegasus
 
 See our documentation on [Pegasus Website][website].
 
+## Client drivers
+
+Pegasus has support for serveral languages:
+
+- [Java](https://github.com/xiaomi/pegasus-java-client) (The most actively-developed)
+- [C++](https://github.com/apache/incubator-pegasus/blob/master/src/include/pegasus/client.h)
+- [Go](https://github.com/xiaomi/pegasus-go-client)
+- [Python](https://github.com/xiaomi/pegasus-python-client)
+- [Node.js](https://github.com/xiaomi/pegasus-nodejs-client)
+- [Scala](https://github.com/xiaomi/pegasus-scala-client)
+
+## Contact us
+
+- Send mails to Apache Pegasus dev mailing list: dev@pegasus.apache.org. This is officially where topics around development, community, and problems are discussed. Please remember to subsribe the mail list via dev-subscribe@pegasus.apache.org.
```

Comment:
> This is officially where topics around development, community, and problems are discussed.

There is something wrong with this sentence.